cmd/crossbuild.go

```diff
@@ -180,9 +172,17 @@ type platformGroup struct {
 func (pg platformGroup) Build(repoPath string) error {
 	if platformsParam := strings.Join(pg.Platforms[:], " "); platformsParam != "" {
 		fmt.Printf("> running the %s builder docker image\n", pg.Name)
```

Comment:
It's idiomatic in go to reduce indentation levels as much as possible. If I unserstand the method correctly, you could do the following:

```go
platformsParam := strings.Join(pg.Platforms[:], " ")
if len(platformsParam) == 0 {
    return nil
}

fmt.Printf("> running the ...")
// ...
```